laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -527,50 +393,31 @@ public function updateMilestones($id)
             ->orderBy("last_name")
             ->get();
 
-        $datas = compact("data1", "data2", "data3");
-
-        return view("schedules.resident.milestone", compact("id", "milestones", "datas", "anesthesiologists"));
+        return view(
+            "schedules.resident.milestone",
+            compact("id", "milestones", "resident_data", "anesthesiologists")
+        );
     }
 
     public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
     {
-        $choice = "";
-        if ($overwrittenChoices[0] == 1) {
-            $choice = "1";
-        }
-
-        if ($overwrittenChoices[1] == 2) {
-            $choice = $choice . " 2";
-        }
-
-        if ($overwrittenChoices[2] == 3) {
-            $choice = $choice . " 3";
-        }
+        $choice = implode(", ", $overwrittenChoices);
+        $choice .= " ";
 
         $subject = "REMODEL: Resident Preference " . $choice . " Overwritten for " . $date;
```

Comment:
To be consistent with the following two lines, what do you think about using string interpolation?